src/compiler/reviews.md

```diff
@@ -127,217 +104,162 @@ bring PRs into good shape and meet the above criteria:
-  `#t-compiler/private` to ask the rest of the team -- for someone who might be
-  able to review it, or even if the team is comfortable with accepting the
-  change at all.
-- If the change is intended for another team, roll a reviewer from the relevant
-  team (example `r? compiler`)
-
-You can also always ask for help on the `#t-compiler` Zulip stream for finding a
-reviewer. That being said, you are always welcome to do an initial review (to
-the extent you are comfortable with) and then pass the PR on to the final
-reviewer. This way the PR author will get helpful feedback sooner, subsequent
-reviewers will have less work to do, and you might also improve your own
-understanding of diverse areas in the compiler.
-
-### It is unclear if something constitutes a major change
-
-Deciding if something is a "major change" requiring an [MCP] is not always
-straightforward. The official guidelines are [here][whats-a-major-change]. When
-in doubt, err on side of treating something as a major change. You can also
-nominate the PR for discussion in the compiler team's triage meeting by tagging
-it `I-compiler-nominated`. If you nominate a PR please make sure to state a
-**concrete question** for the compiler team to discuss, include **useful
-context** if suitable.
-
-> **Example triage meeting nomination message**
-> 
-> ```text
-> Nominating for T-compiler triage meeting to determine if we want to make
-> `-Z unstable-options` also accept value(s), e.g. `-Z unstable-options=values` to
-> guard *unstable values* of *stable flags* like `-C opt-level=4`.
-> 
-> Relevant discussion: <https://i.am.a.link>.
-> See also #123456.
-> 
-> @rustbot label +I-compiler-nominated
-> ```
+It is completely normal that you get (randomly) assigned a PR (via rustbot or otherwise) but don't
+feel comfortable reviewing it. Here is what you can do, depending on the concrete case:
+
+- If the change seems really big or contentious, consider recommending the author go through the
+  [appropriate approval process](./proposals-and-stabilization.md).
+- If you know just the right person for the review, assign them via `r? @<github-name>`. It's polite
+  to leave a comment asking them if they can take over -- but you don't have to make sure
+  beforehand that they can actually do it.
+- If the change is not too complicated and you don't expect that another randomly rolled compiler
+  reviewer will also have trouble with the PR, you can reroll a random compiler reviewer with
+  `r? compiler`.
+- If the change is complicated, or you expect randomly rolling another compiler reviewer will just
+  lead to multiple rerolls, you should open a thread in `#t-compiler/private` to ask the rest of
+  the team -- for someone who might be able to review it, or even if the team is comfortable with
+  accepting the change at all.
+- If the change is intended for another team, roll a reviewer from the relevant team (example:
+  `r? compiler`).
```

Comment:
Nit:

```suggestion
- If the change is intended for another team, roll a reviewer from the relevant team (example:
  `r? libs`).
```

(this is t-compiler docs already 😄)

Reply:
Fixed!